_rules/meta-refresh-no-delay-no-exception-bisz58.md

```diff
@@ -0,0 +1,220 @@
+---
+id: bisz58
+name: '`meta` element has no refresh delay (no exception)'
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used for delayed redirecting or refreshing.
+accessibility_requirements:
+  wcag20:2.2.4: # Interruptions (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:3.2.5: # Change on Request (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G110: # Using an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:H76: # Using meta refresh to create an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+This rule applies to the first `meta` element in a document for which all the following are true:
+
+- the element has an `http-equiv` [attribute value][] of `"refresh"`; and
+- the element has a [valid][valid content] `content` [attribute value][].
```

Comment:
I'm not super happy with the link "valid content" which goes to an **usage** of the `content` term rather than a proper anchor. It's unstable (if usages of `content` are added or removed). But there is no anchor on that specific part of the specs…
Ideas?

Reply:
Having our own definition (that would copy the spec)?
That would take care of the instability issue, but we would need to update the definition if the spec gets updated.